analyses/cell-type-ewings/references/msigdb-gene-sets.tsv

```diff
@@ -0,0 +1,11 @@
+name	geneset	category	subcategory
+staege	STAEGE_EWING_FAMILY_TUMOR	C2	CGP
+miyagawa_up	MIYAGAWA_TARGETS_OF_EWSR1_ETS_FUSIONS_UP	C2	CGP
+miyagawa_down	MIYAGAWA_TARGETS_OF_EWSR1_ETS_FUSIONS_DN	C2	CGP
+zhang	ZHANG_TARGETS_OF_EWSR1_FLI1_FUSION	C2	CGP
+riggi_up	RIGGI_EWING_SARCOMA_PROGENITOR_UP	C2	CGP
+riggi_down	RIGGI_EWING_SARCOMA_PROGENITOR_DN	C2	CGP
+kinsey_up	KINSEY_TARGETS_OF_EWSR1_FLII_FUSION_UP	C2	CGP
+kinsey_down	KINSEY_TARGETS_OF_EWSR1_FLII_FUSION_DN	C2	CGP
+hallmark_EMT	HALLMARK_EPITHELIAL_MESENCHYMAL_TRANSITION	H	

```

Comment:
Can we actually add `NA` for the subcategory here instead of leaving empty? It does get read as `NA` within R, but seems safer to actually write it out here.

Reply:
Yes, I'll change this but then will still keep the change to use NULL in the script.